project/SparkBuild.scala

```diff
@@ -756,12 +756,12 @@ object Assembly {
         .getOrElse(SbtPomKeys.effectivePom.value.getProperties.get("hadoop.version").asInstanceOf[String])
     },
     assemblyJarName in assembly := {
-      lazy val hdpVersion = hadoopVersion.value
+      lazy val versionValue = hadoopVersion.value
```

Comment:
I think `hdp` was the short name of Hadoop instead of "Hortonworks Data Platform" if you meant this but I think it's okay to fix to avoid confusion.

What about we name it `hadoopVersion` instead to make it extra explicit?